# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2024/10
# BioPython 1.83 does not provide protein_letters_3to1 in Bio.Data.SCOPdata but in Bio.Data.PDBData (and Bio.Data.IUPACData)
diff -ru -ru alphafold-2.3.2/alphafold/data/mmcif_parsing.py alphafold-2.3.2_BioPythonSCOPData/alphafold/data/mmcif_parsing.py
--- alphafold-2.3.2/alphafold/data/mmcif_parsing.py	2024-02-19 09:55:16.359778490 +0100
+++ alphafold-2.3.2_BioPythonSCOPData/alphafold/data/mmcif_parsing.py	2023-03-27 13:50:49.000000000 +0200
@@ -21,7 +21,7 @@
 
 from absl import logging
 from Bio import PDB
-from Bio.Data import SCOPData
+from Bio.Data import PDBData as SCOPData
 
 # Type aliases:
 ChainId = str
